components/brave_wallet/browser/eth_json_rpc_controller.cc

```diff
@@ -240,6 +240,41 @@ void EthJsonRpcController::OnGetERC20TokenBalance(
   std::move(callback).Run(true, result);
 }
 
+bool EthJsonRpcController::UnstoppableDomainsProxyReaderGetMany(
+    const std::string& contract_address,
+    const std::string& domain,
+    const std::vector<std::string>& keys,
+    UnstoppableDomainsProxyReaderGetManyCallback callback) {
+  auto internal_callback = base::BindOnce(
+      &EthJsonRpcController::OnUnstoppableDomainsProxyReaderGetMany,
+      base::Unretained(this), std::move(callback));
+  std::string data;
+  if (!unstoppable_domains::GetMany(keys, domain, &data)) {
+    return false;
+  }
+
+  Request(eth_call("", contract_address, "", "", "", data, "latest"),
+          std::move(internal_callback), true);
+  return true;
+}
+
+void EthJsonRpcController::OnUnstoppableDomainsProxyReaderGetMany(
```

Comment:
Could we add some browsertests for these in `browser/brave_wallet/eth_json_rpc_controller_browsertest.cc`?

Reply:
Added in https://github.com/brave/brave-core/pull/8437/commits/732166ca4493d18df5685812f7ade76210d8f71d